cypress/integration/WebInterface/CQL Library/CQLLibrary.ts

```diff
@@ -0,0 +1,19 @@
+import {OktaLogin} from "../../../Shared/OktaLogin"
```

Comment:
This test is fine for now, but likely we will just update this for CQL name validations or something else as navigating to the tab should be inherent to most CQL Library test cases

Reply:
Yeah this is a place holder for CQL Library tests. Will change the names going forward.